backend/src/main/java/com/example/backend/domain/repository/CardJdbcTemplateRepository.java

```diff
@@ -0,0 +1,62 @@
+package com.example.backend.domain.repository;
+
+import com.example.backend.domain.Card;
+import com.example.backend.web.dto.Column;
+import com.example.backend.web.dto.CardMoveRequestDto;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.jdbc.core.simple.SimpleJdbcInsert;
+import org.springframework.stereotype.Repository;
+
+import javax.sql.DataSource;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Optional;
+
+@Repository
+public class CardJdbcTemplateRepository implements CardRepository {
+    private final JdbcTemplate jdbcTemplate;
+    private final SimpleJdbcInsert simpleJdbcInsert;
+
+    public CardJdbcTemplateRepository(DataSource dataSource) {
+        this.jdbcTemplate = new JdbcTemplate(dataSource);
+        this.simpleJdbcInsert = new SimpleJdbcInsert(dataSource)
+                .withTableName("CARD")
+                .usingGeneratedKeyColumns("id");
+    }
+
+    @Override
+    public Column findAllDesc() {
+        return null;
+    }
+
+    @Override
+    public Long save(Card card) {
+        String columnName = card.getColumnName();
+        Integer orderIndex = jdbcTemplate.queryForObject(
+                "SELECT count(*) FROM CARD WHERE column_name = '" + columnName + "'", Integer.class);
+
+        Map<String, Object> params = new HashMap<>();
+        params.put("title", card.getTitle());
+        params.put("content", card.getContent());
+        params.put("column_name", columnName);
+        params.put("author_system", card.getAuthorSystem());
+        params.put("order_index", orderIndex);
+        params.put("deleted", 0);
```

Comment:
```suggestion
        params.put("deleted", false);
```
이렇게는 안 되나요? 👀
만약 가능하다면 가독성이 더 좋아질 것 같네요

Reply:
좀 더 명확하게 boolean의 의미를 가지도록 개선해보겠습니다!